### ISSUE TITLE
sync with letterboxd

### ISSUE THREAD

**Issue by jeyca** [Alcidauk/CineLog#37]
it would be faboulus if we could sync the review and wishlist with letterboxd

**Comment by Alcidauk** [comment]
Hi,

As I said in [this issue](https://github.com/Alcidauk/CineLog/issues/28), the first goal of this app is to keep privacy. That is why I implemented exports: try to keep as much a possible data out of "unknown" cloud solutions.

But I'm curious: what are your use of Cinelog and letterboxd ? Why do you use both of them ?

Thanks for the suggestion,

Pierre

**Comment by sdicke** [comment]
I have another similar proposal. It would be nice to have a synchronisation option for Nextcloud. There are two ways to solve it: Use a plain text file somewhere in the Nexcloud repository or use a Nextcloud app as backend, but that would require PHP development. That is nothing I would call urgent, but nice to have.

**Comment by Korbak** [comment]
Agreed with sdicke. Actually, being able to simply chose the exportation folder would be great.

Here's the dreamt usercase in my opinion:
1. Let the user define the export destination in the settings;
2. A checkbox associated to the daily automated backup allowing to chose to add a timestamp or not (so you can chose to write over the older backup every time) ;

That way, you can just set Cinelog to export in Nextcloud's folder your data without multiplying the files nor manually trigger the export. It's just synchronized every time you open CineLog for the first time of the day (which already is a great included option).

**Comment by Alcidauk** [comment]
Hi @sdicke and @Korbak,

See my complete answer in https://github.com/Alcidauk/CineLog/issues/83 !

Thank you,

Pierre
